### PR TITLE
Added Courses for Big Data 

### PR DESCRIPTION
**Pull request by sashankchittoory**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by sashankchittoory** [comment]
Pls help me find out where checks are failing

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

**Comment by eshellman** [comment]
Put these in data science. 
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
Don't need (YouTube) annotation

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -232,6 +235,21 @@
 * [Shell Scripting Tutorial for Beginners](https://www.youtube.com/playlist?list=PLS1QulWo1RIYmaxcEqw5JhK3b-6rgdWO_) -  ProgrammingKnowledge (YouTube)
 
 
+### Big Data
+
+* [Mastering Big Data Analytics](https://olympus.mygreatlearning.com/courses/10977) - Great Learning
```
Comment (eshellman) [inline courses/free-courses-en.md:240]: not a direct link

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -232,6 +235,21 @@
...
+
+
+#### Hadoop
+
+* [Apache Hadoop Tutorial for Beginners](https://www.youtube.com/watch?v=JK2MdJAWEGc&list=PLlgLmuG_KgbasW0lpInSAIxYd2vqAEPit&ab_channel=GreatLearning) - Great Learning (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:245]: Adding resource in #9177

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -232,6 +235,21 @@
...
+
+
+#### Hive
+
+* [Apache Hive Tutorial](https://www.youtube.com/watch?v=MLDcapDQba4&ab_channel=GreatLearning) - Great Learning (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:250]: Adding resource in #9177

**Review by eshellman — COMMENTED** [review]
Adding resources in #9177

**Pull request closed without merging**